Dockerfile.assisted-service-build-podman

```diff
@@ -0,0 +1,19 @@
+FROM quay.io/podman/stable:latest
+
+ENV GO111MODULE=on
+ENV GOFLAGS=""
+ENV CONTAINER_COMMAND="podman"
+ENV VERSION=1.16     GOCACHE=/go/.cache     GOARM=5     GOPATH=/go     GOROOT=/usr/local/go     GOFLAGS='-mod=vendor'     LOGNAME=deadbeef
+ENV PATH=$PATH:$GOROOT/bin:$GOPATH/bin
+
+RUN dnf install -y libguestfs-tools libvirt-client python3 python3-devel unzip gcc git hostname jq krb5-devel libvirt-devel make nmap-ncat openssl rsync tar wget which zip
+RUN set -x && curl --fail --retry 3 -L https://golang.org/dl/go$VERSION.linux-amd64.tar.gz | tar -C /usr/local -xzf - && go get golang.org/x/tools/cmd/cover golang.org/x/tools/cmd/goimports github.com/tools/godep golang.org/x/lint/golint github.com/openshift/release/tools/gotest2junit github.com/openshift/imagebuilder/cmd/imagebuilder && GO111MODULE=on go get gotest.tools/gotestsum@v0.6.0 && mv $GOPATH/bin/* /usr/bin/ && rm -rf $GOPATH/* $GOPATH/.cache && mkdir $GOPATH/bin && ln -s /usr/bin/imagebuilder $GOPATH/bin/imagebuilder && ln -s /usr/bin/goimports $GOPATH/bin/goimports && curl -L https://github.com/golang/dep/releases/download/v0.5.4/dep-linux-amd64 > /usr/bin/dep && chmod +x /usr/bin/dep
```

Comment:
Why not just:
* `dnf install -y golang`
* setting GOPATH and adding it to PATH
?

Reply:
irrelevant, new solution, it wasn't ready than